src/components/Footer/footer.js

```diff
@@ -1,41 +1,61 @@
 import { BrowserRouter as Router, NavLink } from "react-router-dom";
 import styles from "./footer.module.scss";
+import logo from "../../assets/image/footer/Layer 2.png";
 
 export const Footer = () => {
   return (
     <footer className={styles.footer}>
-      <img className={styles.logo} alt="logo" />
+      <div className={styles.logo_column}>
+        <img className={styles.logo} src={logo} alt="logo" />
```

Comment:
Space between image, first `<p>` and the second `<p>` should be something like 50px.